Code/GraphMol/FileParsers/testMolSupplier.cpp

```diff
@@ -103,6 +104,30 @@ int testMolSup() {
     }
     TEST_ASSERT(i == 16);
   }
+  {
```

Comment:
This block, along with the one below that uses the maeparser, needs an `#ifdef RDK_BUILD_COORDGEN_SUPPORT`

Reply:
Argh.  Sorry, I should really have two build directories (one on, one off, to make sure both works).  I won't commit again w/o checking that way..